resources/content/schema/base/service.json

```diff
@@ -9,7 +9,10 @@
             "type": "int",
             "required": false,
             "nullable": true,
-            "default": 1
+            "default": 1,
+             "attributes" : {
+                "scheduleUpdate" : true
```

Comment:
@cjellick yes. Without this attribute specified, no handler will be invoked on db resource change.